### PR TITLE
SWF-4697 Create project branch for TIM project

### PR DESCRIPTION
**Pull request by madhifallah**

### CODE REVIEW

**Comment by madhifallah** [comment]
Work still in progress, some variable values needs to be identified.
please refer to jira ticket and SWF-4705 for more information.

File: git-workflow/createFB.sh [hunk 1]
```diff
@@ -2,42 +2,49 @@
...
-ADDON_LECKO_TARGET_VERSION=1.3.x-$BRANCH-SNAPSHOT
+# Add-on eXo Lecko not available for 5.1.x
+#ADDON_LECKO_ORIGIN_VERSION=1.3.x-spaces-administration-SNAPSHOT
+#ADDON_LECKO_TARGET_VERSION=1.3.x-$BRANCH-SNAPSHOT
+
+#Add-on Answers
+ADDON_ANSWERS_ORIGIN_VERSION=2.1.2
+ADDON_ANSWER_TARGET_VERSION=2.1.x-$BRANCH-SNAPSHOT
+
+
```
Comment (vsellier) [inline git-workflow/createFB.sh]:
```suggestion
```

File: git-workflow/createFB.sh [hunk 2]
```diff
@@ -2,42 +2,49 @@
...
-ADDON_LECKO_ORIGIN_VERSION=1.3.x-spaces-administration-SNAPSHOT
-ADDON_LECKO_TARGET_VERSION=1.3.x-$BRANCH-SNAPSHOT
+# Add-on eXo Lecko not available for 5.1.x
+#ADDON_LECKO_ORIGIN_VERSION=1.3.x-spaces-administration-SNAPSHOT
+#ADDON_LECKO_TARGET_VERSION=1.3.x-$BRANCH-SNAPSHOT
+
+#Add-on Answers
+ADDON_ANSWERS_ORIGIN_VERSION=2.1.2
+ADDON_ANSWER_TARGET_VERSION=2.1.x-$BRANCH-SNAPSHOT
+
```
Comment (vsellier) [inline git-workflow/createFB.sh]:
```suggestion
```

File: git-workflow/createFB.sh [hunk 3]
```diff
@@ -90,7 +97,7 @@ function replaceProjectVersion() {
 	chat-application) $SCRIPTDIR/../replaceInFile.sh "<version>$ADDON_CHAT_ORIGIN_VERSION</version>" "<version>$ADDON_CHAT_TARGET_VERSION</version>" "pom.xml -not -wholename \"*/target/*\"" ;;
 	exo-es-embedded) $SCRIPTDIR/../replaceInFile.sh "<version>$ADDON_ES_EMBED_ORIGIN_VERSION</version>" "<version>$ADDON_ES_EMBED_TARGET_VERSION</version>" "pom.xml -not -wholename \"*/target/*\"" ;;
 	gatein-dep) $SCRIPTDIR/../replaceInFile.sh "<version>$GATEIN_DEP_ORIGIN_VERSION</version>" "<version>$GATEIN_DEP_TARGET_VERSION</version>" "pom.xml -not -wholename \"*/target/*\"" ;;
-	lecko) $SCRIPTDIR/../replaceInFile.sh "<version>$ADDON_LECKO_ORIGIN_VERSION</version>" "<version>$ADDON_LECKO_TARGET_VERSION</version>" "pom.xml -not -wholename \"*/target/*\"" ;;
+	answers) $SCRIPTDIR/../replaceInFile.sh "<version>$ADDON_ANSWERS_ORIGIN_VERSION</version>" "<version>$ADDON_ANSWERS_TARGET_VERSION</version>" "pom.xml -not -wholename \"*/target/*\"" ;;
```
Comment (vsellier) [inline git-workflow/createFB.sh]: You should let lecko to miss something on the future 5.2.x fb creations.
just add answer

File: git-workflow/createFB.sh [hunk 4]
```diff
@@ -2,42 +2,49 @@
...
-# Add-on eXo Lecko
-ADDON_LECKO_ORIGIN_VERSION=1.3.x-spaces-administration-SNAPSHOT
-ADDON_LECKO_TARGET_VERSION=1.3.x-$BRANCH-SNAPSHOT
+# Add-on eXo Lecko not available for 5.1.x
+#ADDON_LECKO_ORIGIN_VERSION=1.3.x-spaces-administration-SNAPSHOT
+#ADDON_LECKO_TARGET_VERSION=1.3.x-$BRANCH-SNAPSHOT
+
+#Add-on Answers
+ADDON_ANSWERS_ORIGIN_VERSION=2.1.2
+ADDON_ANSWER_TARGET_VERSION=2.1.x-$BRANCH-SNAPSHOT
```
Comment (vsellier) [inline git-workflow/createFB.sh]:
```suggestion
ADDON_ANSWERS_TARGET_VERSION=2.1.x-$BRANCH-SNAPSHOT
```

File: git-workflow/createFB.sh [hunk 5]
```diff
@@ -141,7 +148,7 @@ function replaceProjectAddons() {
 	printf "\e[1;33m# %s\e[m\n" "Modifying add-ons versions in the packaging project POMs ($repo_name) ..."
 
 	$SCRIPTDIR/../replaceInFile.sh "<addon.exo.es.embedded.version>$ADDON_ES_EMBED_ORIGIN_VERSION</addon.exo.es.embedded.version>" "<addon.exo.es.embedded.version>$ADDON_ES_EMBED_TARGET_VERSION</addon.exo.es.embedded.version>" "pom.xml -not -wholename \"*/target/*\""
-	$SCRIPTDIR/../replaceInFile.sh "<addon.exo.lecko.version>$ADDON_LECKO_ORIGIN_VERSION</addon.exo.lecko.version>" "<addon.exo.lecko.version>$ADDON_LECKO_TARGET_VERSION</addon.exo.lecko.version>" "pom.xml -not -wholename \"*/target/*\""
+	$SCRIPTDIR/../replaceInFile.sh "<addon.exo.answers.version>$ADDON_ANSWERS_ORIGIN_VERSION</addon.exo.answers.version>" "<addon.exo.answers.version>$ADDON_ANSWERS_TARGET_VERSION</addon.exo.answers.version>" "pom.xml -not -wholename \"*/target/*\""
```
Comment (vsellier) [inline git-workflow/createFB.sh]: same here, add answers and let lecko.
If it's not present, nothing will be done

File: git-workflow/createFB.sh [hunk 6]
```diff
@@ -2,42 +2,49 @@
...
 # Add-on eXo Web Pack
-ADDON_WEB_PACK_ORIGIN_VERSION=2.2.x-spaces-administration-SNAPSHOT
-ADDON_WEB_PACK_TARGET_VERSION=2.2.x-$BRANCH-SNAPSHOT
+ADDON_WEB_PACK_ORIGIN_VERSION=2.1.2
+ADDON_WEB_PACK_TARGET_VERSION=2.1.x-$BRANCH-SNAPSHOT
 # Add-on eXo Web Conferencing
 ADDON_WEB_CONFERENCING_ORIGIN_VERSION=1.3.x-spaces-administration-SNAPSHOT
 ADDON_WEB_CONFERENCING_TARGET_VERSION=1.3.x-$BRANCH-SNAPSHOT
 # Add-on eXo Push notifications
 ADDON_PUSH_NOTIFICATIONS_ORIGIN_VERSION=1.1.x-spaces-administration-SNAPSHOT
```
Comment (vsellier) [inline git-workflow/createFB.sh]:
```suggestion
ADDON_PUSH_NOTIFICATIONS_ORIGIN_VERSION=1.0.2
```

File: git-workflow/createFB.sh [hunk 7]
```diff
@@ -2,42 +2,49 @@
...
 ADDON_PUSH_NOTIFICATIONS_TARGET_VERSION=1.1.x-$BRANCH-SNAPSHOT
```
Comment (vsellier) [inline git-workflow/createFB.sh]:
```suggestion
ADDON_PUSH_NOTIFICATIONS_TARGET_VERSION=1.0.x-$BRANCH-SNAPSHOT
```

**Pull request closed without merging**